tests/test_other.py

```diff
@@ -0,0 +1,16 @@
+# Standard Python modules
+import sys
+import unittest
+
+# First party modules
+from pyoptsparse.pySNOPT.pySNOPT import _import_snopt_from_path
+
+
+class TestImportSnoptFromPath(unittest.TestCase):
+    def test_nonexistent_path(self):
+        assert _import_snopt_from_path("/a/nonexistent/path") is None
```

Comment:
Maybe directly use `self.assertIsNone` instead.
Also, do we want to test that the warning is emitted here? Should be able to catch by something like `with self.assertWarns(Warning):` (same below?)

Reply:
Done, also changed the warning to be `ImportWarning` which seems appropriate.